### PR TITLE
11270 welcome

### PR DESCRIPTION
**Pull request by cdelacruz8512**

### CODE REVIEW

**Review by juantintos — CHANGES_REQUESTED** [review]
You have a conflict in your files, fix that

**Comment by coderdave** [comment]
Development needs to be merged in.

File: test/mailers/previews/chef_mailer_preview.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Preview all emails at http://localhost:3000/rails/mailers/chef_mailer
+class ChefMailerPreview < ActionMailer::Preview
+    def welcome_email
+      #chef = current_chef
```
Comment (vicmaburrito) [inline test/mailers/previews/chef_mailer_preview.rb:4]: - [ ] Also remove from here

**Review by vicmaburrito — COMMENTED** [review]
Hello Folks! 👋🏽

Good job working on the project so far!
✔️ Your app runs fine.
However, there are some suggestions that you need to work on but you are almost there!

**Comment by cdelacruz8512** [comment]
the branch was fixed, no conflicts to merge

**Review by juantintos — APPROVED** [review]

**Review by ArmandoD97 — APPROVED** [review]

**Review by AlexRS90 — APPROVED** [review]
## Hello team!!
### Good job in this milestone, the project is working as expected and all the functionalities are still working! 
### I agree with @vicmaburrito suggestion, removing your commented code to keep it clean
### I'd like to suggest you to write a description for your PR in order to keep in mind what do you do in this branch!

STATUS: Approved 🚀 🎸 🚀 

Merge time!

**Review by faustino-garcia — APPROVED** [review]
Good job! 

I also think It will be a good idea if you write a description for your PR.

**Comment by ArmandoD97** [comment]
There are several plugins in VSCode that make the documentation for you with your comments

**Pull request merged**